src/test/java/ua/ithillel/lesson4/ArrayValueCalculatorTest.java

```diff
@@ -0,0 +1,51 @@
+package ua.ithillel.lesson4;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ArrayValueCalculatorTest {
+    @Test
+    void test1() throws ArrayDataException {
+        ArrayValueCalculator array = new ArrayValueCalculator();
+        String[][] myArray = {{"5", "7", "1", "9"}, {"5", "8", "1", "7"}, {"6", "2", "1", "10"}, {"17", "8", "11", "12"}};
+        assertEquals(110, array.doCalc(myArray));
+    }
+
+    @Test
+    void test2() throws ArrayDataException {
```

Comment:
```suggestion
    void test2() {
```
можна (навіть бажано) без цього